### PR TITLE
Set  integration test timeout

### PR DESCRIPTION
**Pull request by louiseschmidtgen**
## Description
I would like to add a timeout to the integration test. It is not possible to set it on our side (k8s-operator) and has to be set in the actual integration test job. I propose a timeout of 120 minutes.

The reason for the request is an integration test running for 6 hours and not being able to cancel the workflow: https://github.com/canonical/k8s-operator/actions/runs/8418728595/job/23049966931

@addyess

### CODE REVIEW

**Comment by yanksyoon** [comment]
/canonical/self-hosted-runners/run-workflows c15ce86733ae8ebf69a3e1861f23b3408937f7ad

File: .github/workflows/integration_test_run.yaml [hunk 1]
```diff
@@ -151,6 +151,7 @@ on:
 jobs:
   integration-test:
     name: Integration tests
+    timeout-minutes: 120
```
Comment (addyess) [inline .github/workflows/integration_test_run.yaml:154]: any chance you could make this an input called "test-timeout" or some such?

**Review by addyess — CHANGES_REQUESTED** [review]
Maybe make a test-timeout input?

**Comment by addyess** [comment]
@arturo-seijas and @cbartz, Do you think this is a good addition to the integration tests.  We found some rouge tests running after 5 hours because of failed test tear down.  The default gh run is 6 hours if the timeout isn't specified

**Review by jdkandersson — DISMISSED** [review]

File: .github/workflows/integration_test_run.yaml
Comment (jdkandersson) [inline .github/workflows/integration_test_run.yaml:1]: Oh I forgot to mention, please document this input option in the readme 🙂

Comment (louiseschmidtgen) [reply .github/workflows/integration_test_run.yaml:1]: I will add it now thanks!

**Comment by cbartz** [comment]
/canonical/self-hosted-runners/run-workflows f7fb90ed93d1fec5b3f56a7acd088a39e56f6fa5

**Review by cbartz — APPROVED** [review]

**Comment by addyess** [comment]
@jdkandersson do we have your +1?

**Review by jdkandersson — APPROVED** [review]

**Comment by jdkandersson** [comment]
I think we'll just need someone to help you pass the tests, perhaps @yanksyoon?

**Comment by yanksyoon** [comment]
@jdkandersson This looks good to me, would you be able to merge this until we have a fix for the forked artifact uploads please?

**Pull request merged**